templates/vhost/_ssl.erb

```diff
@@ -1,68 +1,67 @@
-<% if @ssl -%>
-
+<%- if @ssl -%>
   ## SSL directives
   SSLEngine on
   <%- unless @mdomain -%>
   SSLCertificateFile      "<%= @ssl_cert %>"
   SSLCertificateKeyFile   "<%= @ssl_key %>"
   <%- end -%>
   <%- if @ssl_chain -%>
-  SSLCertificateChainFile "<%= @ssl_chain %>"
+    SSLCertificateChainFile "<%= @ssl_chain %>"
```

Comment:
Please keep the indenting consistent throughout this file. It's way more important that the final rendered file looks consistent than that the if/else indenting matches.
```suggestion
  SSLCertificateChainFile "<%= @ssl_chain %>"
```

Reply:
sure

Reply:
done